viewers/nuxeo-video-viewer.html

```diff
@@ -38,27 +39,26 @@
 
       #video {
         background: var(--primary-background-color);
-        height: 100%;
+        max-height: 100%;
         width: 100%;
         outline: none;
         @apply --layout-flex;
       }
 
       #video.hasStoryboard {
-        height: calc(100% - 85px);
+        max-height: calc(100% - 85px);
       }
 
       #storyboard {
-        display: block;
         background: var(--primary-background-color);
+        display: grid;
```

Comment:
This is a hack for a problem that I believe should be fixed on `nuxeo-document-page`: the fact that it allows content to overflow: I'd keep this as `block`:
```suggestion
        display: block;
```
And change the following on `nuxeo-document-page`:
```diff
diff --git a/elements/document/nuxeo-document-page.html b/elements/document/nuxeo-document-page.html
index ec102fe6..cba8254e 100644
--- a/elements/document/nuxeo-document-page.html
+++ b/elements/document/nuxeo-document-page.html
@@ -78,6 +78,7 @@ limitations under the License.
         @apply --layout-vertical;
         @apply --layout-flex-2;
         padding-right: 8px;
+        overflow: hidden;
       }
 
       :host([opened]) .main {
```